app/javascript/upload.js

```diff
@@ -63,6 +63,10 @@ const uploadAll = function(files) {
 
 document.addEventListener('turbolinks:load', function() {
   // Show dropzone-overlay when a file is dragged onto the page
+  if (document.getElementById('dropzone') === null) {
+    return
+  }
```

Comment:
When would this trigger?

Reply:
The admin pages.  I believe all of the JS changes that aren't directly related to action cable or the changed package names were from that.  I saw them when I went checking to make sure they were working properly and noticed them in the console.